src/main/resources/db/migration/R__data_dictionary.sql

```diff
@@ -167,4 +167,6 @@ COMMENT ON TABLE changelog IS 'details about any amendments to an existing refer
 
 COMMENT ON TABLE referral_location IS 'details about the service user''s current location';
 
+COMMENT ON TABLE complexity_level IS 'adding a new column of type Enum to maintain order';
```

Comment:
Data analysts will read these comments to understand what information these fields convey.

They won't be interested in the use of these in sorting. From their perspective, it's more important that these categorical values represent the complexity of a referred pathway

Reply:
updated to a more understandable statement